src/osg/Texture.cpp

```diff
@@ -2083,49 +2380,137 @@ bool Texture::areAllTextureObjectsLoaded() const
     return true;
 }
 
-GLenum Texture::selectSizedInternalFormat(const osg::Image* image) const
```

Comment:
Why was this removed?

Reply:
OK, I think I have it all worked through. I think we want this new function and a few other changes to applyTexImage2D_load. I will see if I can add them all as suggestions in the comments